tests/Gaufrette/Functional/adapters/AzureMultiContainerBlobStorage.php.dist

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
Thank you :) I will set up Azure Storage to ensure tests against code landing on our repository tests are are run.